### ISSUE TITLE
Audio Muting: AMC+

### ISSUE THREAD

**Issue by richardfrost** [FrostCo/AdvancedProfanityFilter#478]
#### :movie_camera: The Site
https://www.amcplus.com/

#### :speech_balloon: Subtitle Element/Node
_Example:_
```html
<div class="vjs-text-track-display" translate="yes" aria-live="off" aria-atomic="true">
  <div style="position: absolute; inset: 0px; margin: 1.5%;">
    <div class="vjs-text-track-cue vjs-text-track-cue-en" lang="en" style="direction: ltr; writing-mode: horizontal-tb; unicode-bidi: plaintext; text-align: center; font: 56.32px sans-serif; white-space: pre-line; position: absolute; width: 1860.47px; inset: 996.469px 0px 0px; height: 130px; background-color: rgba(0, 0, 0, 0);">
      <div style="color: rgb(255, 255, 255); background-color: rgb(0, 0, 0); position: relative; inset: 0px; display: inline; writing-mode: horizontal-tb; unicode-bidi: plaintext; font-family: sans-serif;">
        What do you mean,
        you saw him?
      </div>
    </div>
  </div>
</div>
```

**Comment by richardfrost** [comment]
Suggested config:

```json
{
  "www.amcplus.com": [
    {
      "mode": "element",
      "subtitleSelector": "div.vjs-text-track-cue > div",
      "tagName": "DIV"
    }
  ]
}
```

**Comment by richardfrost** [comment]
This has been included in [version 2.40.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.40.0). Feel free to comment here if you have any feedback for this update.
